spec/js_routes/rails_routes_compatibility_spec.rb

```diff
@@ -115,8 +115,13 @@
     it "should support route with parameters" do
       expect(evaljs("Routes.blog_app_post_path(1)")).to eq(blog_routes.post_path(1))
     end
-    it "should support route with parameters containing symbols that need URI-encoding" do
+    it "should support route with parameters containing symbols that need URI-encoding", :aggregate_failures do
       expect(evaljs("Routes.blog_app_post_path('#hello')")).to eq(blog_routes.post_path('#hello'))
+      expect(evaljs("Routes.blog_app_post_path('some param')")).to eq(blog_routes.post_path('some param'))
+    end
+    it "should support route with parameters containing symbols not need URI-encoding", :aggregate_failures do
+      expect(evaljs("Routes.blog_app_post_path(':some_id')")).to eq(blog_routes.post_path(':some_id'))
+      expect(evaljs("Routes.blog_app_post_path('.+')")).to eq(blog_routes.post_path('.+'))
```

Comment:
The routes from mounted rails engine are used here. I don't think this is good idea because the thing we actually test can be done with a simplier route which will cause less side effects from possible bugs in engines support.